src/layer/ImageOverlay.js

```diff
@@ -53,7 +53,14 @@ export const ImageOverlay = Layer.extend({
 
 		// @option className: String = ''
 		// A custom class name to assign to the image. Empty by default.
-		className: ''
+		className: '',
+
+		// @option syncDecoding: Boolean = true
+		// Tells the browser to [decode the image in a synchronous fashion](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding),
+		// in order to reduce image flickering under some workloads. It should be
+		// set to `false` whenever the image overlay is not expected to be
+		// removed/readded to the map.
+		syncDecoding: true

```

Comment:
Perhaps we can use the [same values](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding#value) here as the attribute supports? That way the user could also decide to go for `auto`?